.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,50 +1,52 @@
 ---
 name: Bug report
 about: Found a bug in Fabric? Please let us know.
-
 ---
-<!--  
+
+<!--
 Thanks for contacting us! We're here to help.
 
 Before you report an issue, check if it's been reported before:
 
   * Search: https://github.com/OfficeDev/office-ui-fabric-react/search?type=Issues
   * Search by area or component: https://github.com/OfficeDev/office-ui-fabric-react/issues/labels
-  
+
 Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
 
-  * Goto https://aka.ms/fabricpen for a starting codepen 
+  * Goto https://aka.ms/fabricpen for a starting codepen
   * You can also use the "Export to Codepen" feature for the various components in our documentation site.
-  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example. 
+  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example.
     (A permanent link is preferable to "use the website" as the website can change)
- 
+
 Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+
+Also note that this Github repository is a public forum.
```

Comment:
Maybe move to the beginning and say something like:

> WARNING: This GitHub repository is a public forum. DO NOT share links or info which should not be available on the public internet: i.e. internal issue tracking links, screenshots of products in development, etc.

But as you pointed out elsewhere, they can still ignore it regardless of all our efforts...

Reply:
Or closer to the beginning at least.

Reply:
Links are typically ok, assuming the URL doesn't reveal anything innapropriate.